### PR TITLE
[W5.11][W14-A3]Randy Pang

### PR DESCRIPTION
**Pull request by randypx**
Added two features for address book. 
A sort feature to display all persons in the address book in alphabetical order
And a edit feature to change the stored parameters of a person specified by the index number in the last shown list.

### CODE REVIEW

File: src/seedu/addressbook/commands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    private int targetIndex;
+    private final Person toEdit;
+
+    /**
+     * Convenience constructor using raw values.
+     *
+     * @throws IllegalValueException if any of the raw values are invalid
+     */
+    public EditCommand(int index, String name,
```
Comment (randypx) [inline src/seedu/addressbook/commands/EditCommand.java]: constructor creates a person object from the parameters given.
This newly created person then replaces the person stored at the target index during execute().

File: src/seedu/addressbook/commands/EditCommand.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Person edited: %1$s";
+
+    private int targetIndex;
+    private final Person toEdit;
+
+    /**
+     * Convenience constructor using raw values.
+     *
+     * @throws IllegalValueException if any of the raw values are invalid
+     */
```
Comment (randypx) [inline src/seedu/addressbook/commands/EditCommand.java:35]: constructor creates a person object from the parameters given.
This newly created person then replaces the person stored at the target index during execute().

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.TreeSet;
```
Comment (randypx) [inline src/seedu/addressbook/data/person/UniquePersonList.java]: TreeSet is not used here. I had forgot to delete this import

Comment (kowshik-sundararajan) [reply src/seedu/addressbook/data/person/UniquePersonList.java]: Do remember to remove it, you can update the PR through a cleanup commit.

**Review by kowshik-sundararajan — COMMENTED** [review]
Good job! 👍 Suggestion for future is to submit different PRs for each feature unless they are dependent on each other.

Some comments added. Please close the PR after reading them.

**Pull request closed without merging**